src/components/authorization/policies/by-feature/engagements-create-delete.policy.ts

```diff
@@ -3,7 +3,7 @@ import { field, Hierarchies, member, Policy } from '../util';
 @Policy([...Hierarchies.Field, ...Hierarchies.Finance], (r) => [
   r.Project,
   r.Engagement.read,
-  r.Engagement.when(member).create,
+  r.Engagement.whenAll(member, field('project.status', 'InDevelopment')).create,
```

Comment:
Path traversal is so cool! It simplifies a lot the writing and reading of these policies!